content/pages/about/xsf/bylaws.md

```diff
@@ -9,19 +9,25 @@ Dropdown_menu_show: false
 Footer_show: false
 Sidebar_menu_show: true
 Sidebar_menu_title: About the XSF
-Sidebar_menu_size: 6
+Sidebar_menu_size: 9
 Sidebar_menu_elem_name_1: The XSF
 Sidebar_menu_elem_url_1: about/xmpp-standards-foundation
 Sidebar_menu_elem_name_2: Members
 Sidebar_menu_elem_url_2: about/xsf/members
-Sidebar_menu_elem_name_3: Editor Team
-Sidebar_menu_elem_url_3: about/xsf/editor-team
-Sidebar_menu_elem_name_4: Infrastructure Team
-Sidebar_menu_elem_url_4: about/xsf/infrastructure-team
-Sidebar_menu_elem_name_5: UPnP Liaison Team
-Sidebar_menu_elem_url_5: about/xsf/upnp-liaison-team
-Sidebar_menu_elem_name_6: Standards Process
-Sidebar_menu_elem_url_6: about/standards-process
+Sidebar_menu_elem_name_3: Communication Team
+Sidebar_menu_elem_url_3: about/xsf/communications-policy
+Sidebar_menu_elem_name_4: Editor Team
+Sidebar_menu_elem_url_4: about/xsf/editor-team
+Sidebar_menu_elem_name_5: Infrastructure Team
+Sidebar_menu_elem_url_5: about/xsf/infrastructure-team
+Sidebar_menu_elem_name_6: Summits, Conferences & Meetups Team
+Sidebar_menu_elem_url_6: about/xsf/scam-team
+Sidebar_menu_elem_name_7: Technical Review Team
```

Comment:
Stillborn. I think it only ever managed to review one or two, and then stalled before it got started.